core/src/main/java/lucee/runtime/tag/Schedule.java

```diff
@@ -579,7 +579,7 @@ private String printUrl(URL url) {
 		if (qs == null) qs = "";
 		else if (qs.length() > 0) qs = "?" + qs;
 
-		String str = url.getProtocol() + "://" + url.getHost() + url.getPath() + qs;
+		String str = url.getProtocol() + "://" + url.getHost() + ":" + Caster.toString(HTTPUtil.getPort(url)) + url.getPath() + qs; 
```

Comment:
@cfmitrah let's only add the port if non standard, i.e. not 443 with https or 80 with http?

Reply:
@zspitzer I've modified the fix to append the port if it is non-standard and also added the testcase